test/integration/test_containerized_jobs.py

```diff
@@ -109,6 +116,10 @@ def test_build_mulled(self):
         assert any([True for d in response if d['dependency_type'] == self.container_type])
 
 
+# Singularity 2.4 in the official Vagrant issue has some problems running this test
```

Comment:
@jmchilton for your information, we have generated all 32.000 Singularity images with Singularity 3.x. This means they are not compatible with Singularity 2.x.

Reply:
Good to know, thanks. I need a better test setup for Singularity for sure - luckily all the problems I landed up encountering with Pulsar actually were for all containers 😆.